packages/layer-composer/src/generators/heatmap/util/index.ts

```diff
@@ -6,3 +9,15 @@ export const toURLArray = (paramName: string, arr: string[]) => {
     })
     .join('&')
 }
+
+export const getSourceId = (config: GlobalHeatmapAnimatedGeneratorConfig, timeChunk: TimeChunk) => {
+  return `${config.id}-${timeChunk.id}`
+}
+
+export const getLayerId = (
```

Comment:
I love it, thanks for create it! 
However I'd like to talk about normalise the separators, because for example in [context-layers](https://github.com/GlobalFishingWatch/frontend/blob/fourwings-aggregate/env-layers/packages/layer-composer/src/generators/context/context-layers.ts#L104) it is using `_` instead of '-' which ends up having a hard to read sources /layers list. So let's choose one and use it everywhere 🙏

Reply:
Go for `_`